packages/react-core/src/components/Panel/examples/Panel.md

```diff
@@ -0,0 +1,181 @@
+
+const BasicPanel = () => (
+  <Panel>
+    <PanelMain>
+      <PanelMainBody>Main content</PanelMainBody>
+    </PanelMain>
+  </Panel>
+);
+```
+
+### Header
+
+```js
+import React from 'react';
+import { Panel, PanelMain, PanelMainBody, PanelHeader, Divider } from '@patternfly/react-core';
+
+const HeaderPanel = () => (
+  <Panel>
+    <PanelHeader>Header content</PanelHeader>
+    <Divider />
+    <PanelMain>
+      <PanelMainBody>Main content</PanelMainBody>
+    </PanelMain>
+  </Panel>
+);
+```
+
+### Footer
+
+```js
+import React from 'react';
+import { Panel, PanelMain, PanelMainBody, PanelFooter } from '@patternfly/react-core';
+
+const FooterPanel = () => (
+  <Panel>
+    <PanelMain>
+      <PanelMainBody>Main content</PanelMainBody>
+    </PanelMain>
+    <PanelFooter>Footer content</PanelFooter>
+  </Panel>
+);
+```
+
+### Header and footer
+
+```js
+import React from 'react';
+import { Panel, PanelMain, PanelMainBody, PanelHeader, Divider, PanelFooter } from '@patternfly/react-core';
+
+const HeaderFooterPanel = () => (
+  <Panel>
+    <PanelHeader>Header content</PanelHeader>
+    <Divider />
+    <PanelMain>
+      <PanelMainBody>Main content</PanelMainBody>
+    </PanelMain>
+    <PanelFooter>Footer content</PanelFooter>
+  </Panel>
+);
+```
+
+### No body
+
+```js
+import React from 'react';
+import { Panel, PanelMain } from '@patternfly/react-core';
+
+const NoBodyPanel = () => (
+  <Panel>
+    <PanelMain>Main content</PanelMain>
+  </Panel>
+);
+```
+
+### Raised
+
+```js
+import React from 'react';
+import { Panel, PanelMain, PanelMainBody } from '@patternfly/react-core';
+
+const RaisedPanel = () => (
+  <Panel isRaised>
+    <PanelMain>
+      <PanelMainBody>Main content</PanelMainBody>
+    </PanelMain>
+  </Panel>
+);
+```
+
+### Bordered
+
+```js
+import React from 'react';
+import { Panel, PanelMain, PanelMainBody } from '@patternfly/react-core';
+
+const BorderedPanel = () => (
+  <Panel isBordered>
+    <PanelMain>
+      <PanelMainBody>Main content</PanelMainBody>
+    </PanelMain>
+  </Panel>
+);
+```
+
+### Scrollable
+
+```js
+import React from 'react';
+import { Panel, PanelMain, PanelMainBody } from '@patternfly/react-core';
+
+const ScrollablePanel = () => {
+  const mainContent = (
+    <>
+      Main content
+      <br />
+      <br />
+    </>
+  );
+
+  return (
+    <Panel isScrollable>
+      <PanelMain>
+        <PanelMainBody>
+          <>
+          {Array(8).fill(mainContent)}
+          Main content
+          </>
+        </PanelMainBody>
+      </PanelMain>
+    </Panel>
+  );
+};
+```
+
+### Scrollable with header and footer
+
+```js
+import React from 'react';
+import { Panel, PanelMain, PanelMainBody, PanelHeader, Divider, PanelFooter } from '@patternfly/react-core';
+
+const ScrollableHeaderFooterPanel = () => {
+  const mainContent = (
+    <>
+      Main content
+      <br />
+      <br />
+    </>
+  );
+
+  return (
+    <Panel isScrollable>
```

Comment:
We need to add a tab index so that the scrollable content is accessible via keyboard.

cc @jessiehuff